daml-lf/encoder/src/main/scala/com/digitalasset/daml/lf/archive/testing/EncodeV1.scala

```diff
@@ -765,17 +764,10 @@ private[daml] class EncodeV1(minor: LV.Minor) {
 
   }
 
-  private def versionIsOlderThan(minVersion: LV): Boolean =
```

Comment:
why don’t you like those anymore? I don’t have an issue with removing them just curious

Reply:
Now that I have proper odering on `LanguageVersion` I found the the comparison with `<` or `<=`  more readable. 
This is come from a time where I have to compare version with `v1.precedes(v2)`.
(see other comments)